main.go

```diff
@@ -174,12 +175,18 @@ func createRedisClient() (redis.UniversalClient, error) {
 		return nil, errors.Wrap(err, "parsing redis url")
 	}
 
-	// Create a Redis client
-	client := redis.NewUniversalClient(&redis.UniversalOptions{
+	clientOptions := redis.UniversalOptions{
 		Addrs:    []string{parsedRedisURL.Addr},
 		DB:       parsedRedisURL.DB,
 		Password: parsedRedisURL.Password,
-	})
+	}
+
+	if config.RedisTLS() {
+		clientOptions.TLSConfig = &tls.Config{}
```

Comment:
@jgdef-tulip  says add MinVersion, MaxVersion 1.2,1.3

Reply:
InsecureSkipVerify =false

Reply:
Hmm do we actually need to specify maxversion? Removed from now so it will default to latest TLS version